sdk/python/pulumi_aws/directconnect/get_location.py

```diff
@@ -107,7 +107,7 @@ def get_location(location_code: Optional[str] = None,
     Retrieve information about a specific AWS Direct Connect location in the current AWS Region.
     These are the locations that can be specified when configuring `directconnect.Connection` or `directconnect.LinkAggregationGroup` resources.
 
-    > **Note:** This data source is different from the `directconnect.get_locations` data source which retrieves information about all the AWS Direct Connect locations in the current AWS Region.
+    > **Note:** This data source is different from the _directconnect_get_locations_ data source which retrieves information about all the AWS Direct Connect locations in the current AWS Region.
```

Comment:
This change from code to emphasis is all over the Python SDK. I don't think this will render correctly if the inner `_` are not escaped. It also doesn't seem particularly better? Where does this come from?

Reply:
Given that this is a change only in specific languages, my guess is it is a codegen change. I agree that it doesn't look like a positive change.

Reply:
Agreed. This is likely incorrect.

Reply:
I opened a ticket on it: https://github.com/pulumi/pulumi/issues/12056

Reply:
linking to https://github.com/pulumi/pulumi/issues/12075